std/internal/math/gammafunction.d

```diff
@@ -531,8 +548,17 @@ unittest {
 
 
 private {
-enum real MAXLOG = 0x1.62e42fefa39ef358p+13L;  // log(real.max)
-enum real MINLOG = -0x1.6436716d5406e6d8p+13L; // log(real.min*real.epsilon) = log(smallest denormal)
+static if (floatTraits!(real).realFormat == RealFormat.ieeeExtended) {
+    enum real MAXLOG = 0x1.62e42fefa39ef358p+13L;  // log(real.max)
+    enum real MINLOG = -0x1.6436716d5406e6d8p+13L; // log(real.min_normal*real.epsilon) = log(smallest denormal)
+}
+else static if (floatTraits!(real).realFormat == RealFormat.ieeeDouble) {
+    enum real MAXLOG = 0x1.62e42fefa39efp+9L;  // log(real.max)
+    enum real MINLOG = -0x1.74385446d71c3p+9L; // log(real.min_normal*real.epsilon) = log(smallest denormal)
```

Comment:
Smallest denormal  is `2^-1074`, however CEPHES uses `2^-1075` (probably because rounding rule?). Lets it be `2^-1074` ;)

[CEPHES github fork fo reference](https://github.com/jeremybarnes/cephes/blob/60f27df395b8322c2da22c83751a2366b82d50d1/cmath/const.c) (origin has the same constant).

Wolfram:
1. [Example] (http://www.wolframalpha.com/input/?i=hexadecimal+log%282%5E1075%29+)
2. D code example for transformation `writefln("%a", 0x2e9.221aa5a60a3p+0);`